svf/lib/AbstractExecution/AE-SVFIR/AE.cpp

```diff
@@ -444,7 +444,7 @@ bool AE::hasSwitchBranchES(const SVFVar* var, s64_t succ, IntervalExeState& es)
     }
     while(!workList.empty()) {
         const SVFStmt* stmt = workList.pop();
-        if (const CopyStmt *copy = SVFUtil::dyn_cast<CopyStmt>(stmt)) {
+        if (SVFUtil::dyn_cast<CopyStmt>(stmt)) {
```

Comment:
SVFUtil::isa<CopyStmt>(stmt)

and fix all others below

Reply:
thanks. it has been fixed.